r2-perf-test/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
   compile project (':r2-sample')
   compile project (':pegasus-common')
   compile project (':test-util')
+  implementation project(path: ':restli-netty', configuration: 'shadow')
```

Comment:
Why does `r2-perf-test` add a dependency here? Did it depend on Netty before?

Reply:
It depends on Netty. Previously this came through r2-netty, but with the shadow plugin this needs to be explicitly placed here now.